xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -834,7 +805,7 @@ struct UserProfileDetailModel: ModelProtocol {
         var actions: [UserProfileDetailAction] = [
             .presentFollowSheet(false),
             .presentFollowNewUserFormSheet(false),
-            .refresh(forceSync: true),
+            .refresh(forceSync: false),
```

Comment:
I'm not quite there _yet_ but soon we'll be able to drop this `forceSync` concept altogether, it's confusing and I would rather react to notifications at the app level in a controlled manner to trigger sync.